### PR TITLE
Document removed support for TS 2.9

### PR DESCRIPTION
**Pull request by sandersn**
I removed support for 2.9 at the same time as the 3.9 release. This updates the documentation to reflect that.

### CODE REVIEW

**Comment by sandersn** [comment]
@saschanaz @galkin @kingwl mind taking a look at this too? Note that only Chinese has the longer section on how to add `typesVersions` support, so the diff looks a bit different.

**Comment by SimonSchick** [comment]
@sandersn Can you link and/or pink the version deprecation schedule?

File: README.ko.md [hunk 1]
```diff
@@ -315,9 +315,9 @@ NPM 의 패키지들은 수시간 안에 갱신될 겁니다. 만약 24 시간
 자료형 정의(Type definition)가 맞는 경우에는 자료형 정의(Type definition)을 수정하지 마세요.
 NPM 패키지의 경우, `node -p 'require("foo")'` 가 원하는 값이라면 `export =` 이 맞고, `node -p 'require("foo").default'` 이 원하는 값이라면 `export default` 이 맞습니다.
 
-#### 자료형 선언(Type declaration)에서 타입스크립트(TypeScript) 2.1 이상의 기능을 사용하고 싶습니다.
+#### 자료형 선언(Type declaration)에서 타입스크립트(TypeScript) 3.1 이상의 기능을 사용하고 싶습니다.
 
-정의(Definition) 머릿주석(`// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`) 뒤에 `// TypeScript Version: 2.1` 를 추가해주시면 됩니다.
+정의(Definition) 머릿주석(`// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`) 뒤에 `// TypeScript Version: 3.1` 를 추가해주시면 됩니다.
```
Comment (saschanaz) [inline README.ko.md:320]: Wow, this part was so outdated 😆

**Review by saschanaz — APPROVED** [review]

**Review by armanio123 — APPROVED** [review]

**Review by Kingwl — APPROVED** [review]
Looks good!

**Review by galkin — APPROVED** [review]

**Review by SimonSchick — APPROVED** [review]

**Comment by sandersn** [comment]
@SimonSchick the schedule is juuust off-screen in this diff. Expand the lines below the first section and you will see it. I didn't have to change it here because it already had 4.0, and I want to leave in old versions like 2.8 and 2.9 so people will know when they stopped being supported.

**Pull request merged**